src/exodus/test/walletmodels_tests.cpp

```diff
@@ -215,22 +205,58 @@ BOOST_AUTO_TEST_CASE(sigma_mint_unequality)
 
     BOOST_CHECK_NE(left, right);
 
-    // Chain state.
+    // Seed Id
+    right = left;
+    right.seedId = Hash160(one);
+
+    BOOST_CHECK_NE(left, right);
+
+    // Serial Id
+    right = left;
+    right.serialId = Hash160(one);
+
+    BOOST_CHECK_NE(left, right);
+
+    // Spend Tx
+    right = left;
+    right.spendTx = uint256();
+
+    BOOST_CHECK_NE(left, right);
+
+    // Block
     right = left;
-    right.chainState.Clear();
+    right.chainState.block = 501;
```

Comment:
We should let chain state comparison to handle by itself so in the future it when there is more fields it will handle automatically.